src/agent/rustjail/src/lib.rs

```diff
@@ -523,6 +523,31 @@ mod tests {
         };
     }
 
+    // Parameters:
+    //
+    // 1: expected Result
+    // 2: actual Result
+    // 3: string used to identify the test on error
+    #[macro_export]
+    macro_rules! assert_result {
```

Comment:
Note to self: we need to see if we can create a local crate that exports this macro so we can use the single definition everywhere.

Reply:
While you were out, we did get the `assert_result` macro into a shared place for the agent, but unfortunately like you said this isn't usable in rustjail. But, it was a slight improvement.

https://github.com/kata-containers/kata-containers/pull/4104